### DISCUSSION TITLE
YimMenu might be detected

### DISCUSSION THREAD

**Discussion by Yimura** [YimMenu/YimMenu#2126] [Announcements]
I've been seeing some reports on different places of the internet of YimMenu being detected.

The below issue is being used to collect data around the bans happening.
https://github.com/YimMenu/YimMenu/issues/2124

Any off-topic talk on that issue will be deleted, please follow the template given in the initial comment.
Off-topic talk can be done on this discussion.

As of now none of the developers have received any bans on their accounts so we're unable to confirm if the menu has been detected fully or if it's a specific feature causing the bans to occur.

**Comment by ghost** [comment]
Does it matter at what point yimmenu is injected (title screen, story mode, online mode) with regard to possible detection?

**Comment by iDominiqe** [comment]
![image](https://github.com/YimMenu/YimMenu/assets/115355282/a4464dfa-0b49-4cbe-b223-3f01a49787cf)

**Comment by lonelybud** [comment]
I also wonder if bringing a player via teleport can also be detected by anti-cheat?
Also if we spam a native method such `ATTACH_ENTITY_TO_ENTITY` a number of times in a very short time i can see that game crashes but can it be like detected by anti-cheat.
small features like removing wanted level from network or player is it detectable?

**Reply by alyxme** [reply]
I tried to express what could possibly trigger the anti-cheat in #2134 and I think that all this is possible. But it is not exactly. Since the leak of the source code, many patches have been released, only the developers know what has been added for detection. Even if they said that 95% of detection was deleted, I would not believe them at all after all. We can only guess through trial and errors. Unfortunately, sometimes, with well-known sad consequences.

**Reply by lonelybud** [reply]
thanks I saw the discussion you mentioned. I will believe that we should try more to play game as it is intended even though the developers know how to fix the detections. Using only bare minimum whenever possible is gonna be effective.

**Comment by LintStalker** [comment]
So I think it's been over a month since it was first detected.  I was using Yimmnenu for a few days after it was detected, and that account hasn't been banned.  So maybe it's ok?  

Is there an ETA on when you will start posting the nightly build dll again?

**Reply by Yimura** [reply]
I'm guessing it should be about time to make the downloads available publicly again.

It has only been 2 weeks btw:
https://github.com/YimMenu/YimMenu/issues/2425

**Reply by LintStalker** [reply]
2 weeks since  the download was available, or 2 weeks since it was detected?  I thought the 2nd detection was around the end of October?  That's when I stopped using it.

Thanks for posting the download.  It's a really awesome mod and I have missed it.
